source/app.css

```diff
@@ -0,0 +1,53 @@
+cloudflare-app[app-id="a-better-browser"] {
```

Comment:
You're correct, users will have to update their selectors to match the new elements.
It's a bit of a chore though using prefixed elements gets a balance of compatibility and keeps broad stylesheets from taking too much effect on the app.